packages/xchain-bitcoin/src/client.ts

```diff
@@ -41,28 +42,46 @@ class Client implements BitcoinClient, XChainClient {
   private phrase = ''
   private sochainUrl = ''
   private blockstreamUrl = ''
+  private rootDerivationPaths: RootDerivationPaths
 
   /**
    * Constructor
    * Client is initialised with network type
    *
    * @param {BitcoinClientParams} params
    */
-  constructor({ network = 'testnet', sochainUrl, blockstreamUrl, phrase }: BitcoinClientParams) {
+  constructor({
+    network = 'testnet',
+    sochainUrl = 'https://sochain.com/api/v2',
+    blockstreamUrl = 'https://blockstream.info',
+    rootDerivationPaths = {
+      mainnet: `m/44'/84'/0'/0'/0/`,
+      testnet: `m/44'/84'/1'/0'/0/`,
+    },
+    phrase,
+  }: BitcoinClientParams) {
     this.net = network
-    this.setSochainUrl(sochainUrl || this.getDefaultSochainUrl())
-    this.setBlockstreamUrl(blockstreamUrl || this.getDefaultBlockstreamUrl())
+    this.rootDerivationPaths = this.fixRootPAths(rootDerivationPaths)
+    this.setSochainUrl(sochainUrl)
+    this.setBlockstreamUrl(blockstreamUrl)
     phrase && this.setPhrase(phrase)
   }
 
-  /**
-   * Get the default sochain url.
-   *
-   * @returns {string} the default sochain url
-   */
-  getDefaultSochainUrl = (): string => {
-    return 'https://sochain.com/api/v2'
+  fixRootPAths(rootDerivationPaths: RootDerivationPaths) {
```

Comment:
yeah i did this at the very beginning of the refactor, when i thought all paths would be compliant with bip44... in retrospect, i will default the paths to be backwards compatible (so old wallets will still work) 
```
   rootDerivationPaths = {
      mainnet: `84'/0'/0'/0/`,
      testnet: `84'/1'/0'/0/`,
    },
```

but i wil NOT automatically remove the m/44'/ since anyone who passes that in should be respected